### ISSUE TITLE
Bootstraper nodes for js-ipfs

### ISSUE THREAD

**Issue by flyingzumwalt** [ipfs/in-web-browsers#30]
This is from the original sprint notes. @diasdavid @whyrusleeping can you clarify? Might have already been addressed in sprint ipfs/pm#310

**Comment by diasdavid** [comment]
Currently, we have some bootstraper nodes with websockets deployed by @lgierth, however we don't have this 100% working from the browser, the tasks to be done are:

- Figure out why the browser can't use websockets to go-ipfs (it can just fine to js-ipfs and I believe that is simply because go-ipfs is not doing an http upgrade)
- Make sure that there is tests for this, dialing on websockets requires DNS + HTTPS, which forces the node to be behind a SSL termination, which might freak out Identify. Ideally we should have the options to announce and not announce some addrs //cc @lgierth

**Comment by flyingzumwalt** [comment]
depends on #34 (or some of its dependencies) to really work

**Comment by lgierth** [comment]
> which forces the node to be behind a SSL termination, which might freak out Identify.

Dialing a /wss addr to a node which thinks it's /ws work fine between go-ipfs and go-ipfs. Explicit announced addresses are still quasi-required for peer routing, but for explicit dialing we're good. Not sure about the js-ipfs side of this.

**Comment by flyingzumwalt** [comment]
this is a duplicate of #34
